tasks/main.yml

```diff
@@ -24,6 +24,51 @@
   when:
     - network_state is defined
     - ansible_distribution_major_version | int < 8
+
+- name: Check if updates for network packages are available through the DNF
+    package manager due to wireless or team interfaces
+  dnf:
+    update_cache: true
+    name: "{{ network_packages }}"
+    state: latest  # noqa package-latest
+  register: dnf_package_update_info
+  check_mode: true
+  when:
+    - ansible_distribution == 'Fedora' or
+      ansible_distribution_major_version | int > 7
+    - __network_wireless_connections_defined
+      or __network_team_connections_defined
```

Comment:
```suggestion
      or __network_team_connections_defined
    - not __network_is_ostree
```

Reply:
It is fine to add it, but I think that update cache itself will only change the `/var/cache/dnf` directory but not the `/usr` directory.